### ISSUE TITLE
Is it possible drag and drop?

### ISSUE THREAD

**Issue by javiavid** [Quorafind/Obsidian-Big-Calendar#18]
I have visited https://github.com/jquense/react-big-calendar and with the addon option of the drop-down menu it is possible to use drag and drop.

Is it currently possible to do this in Obsidian Big Calendar?

**Comment by Quorafind** [comment]
Of course! But now I have to make the plugin pass the review from Licat, so will be available in  0.3.0

**Comment by javiavid** [comment]
Great!

**Comment by javiavid** [comment]
Thanks for fixing it, but the latest version 1.0.0 gives error when installing from obsidian.

![Image](https://github.com/user-attachments/assets/82d70ef2-9127-47b5-8498-0ad7c3261426)
